### ISSUE TITLE
Differences in implantation from original network

### ISSUE THREAD

**Issue by foromer4** [rpautrat/SuperPoint#81]
Hi, Could you please explain a bit about the differences in implantation compared to the original impl.? For example In this impl. I don't see usage of RELU after each con. layer. Instead I see usage of kernal regularisation and batch normalization.
Would be very interested to better understand the motivation for these 
(and maybe other?) differences.
 thanks a lot, Omer

**Comment by rpautrat** [comment]
Hi,

ReLu is actually used in the method tf.layers.conv2d (in the parameter "activation"). I added the kernel regularisation

**Comment by graulef** [comment]
If I might jump into this discussion, I would be interested in adding regularization. What values did you test already for the `kernel_reg` parameter and for what use-cases did you actually see an improvement?

**Comment by rpautrat** [comment]
I think I tried different values between 0.1 and 0.001, but  adding regularization doesn't really improve the performance. However doing an early stopping (I stop training as soon as the metrics stop increasing) helped a bit.
